doc/General/Contributing.md

```diff
@@ -131,9 +136,7 @@ project.
       ```ssh
       git subtree add --squash --prefix=lib/jquery-bootgrid https://github.com/rstaib/jquery-bootgrid.git master
       ```
-    - Add the code to integrate it in a separate commit.  Include:
-        - code to update it in Makefile
-	- Scrutinizer exclusions to .scrutinizer.yml (not needed if added to lib/ folder).
+    - Add the code to integrate it in a separate commit.
```

Comment:
The problem is the commits for updating /vendor can be a little tricky.  It is nice to separate them from your actual code changes (but keep in the same PR)